third_party/move/move-compiler-v2/transactional-tests/tests/control_flow/binop_eval_order.exp

```diff
@@ -0,0 +1,7 @@
+comparison between v1 and v2 failed:
+= processed 2 tasks
+= 
+= task 1 'run'. lines 9-9:
+- return values: 4
```

Comment:
This is a semantics change from v1. We need to decide whether this change is acceptable.

Reply:
Update the code to keep the behavior of v1